pages.pl/linux/apt-moo.md

```diff
@@ -0,0 +1,8 @@
+# apt moo
+
+> Jajo wielkanoce `APT`.
+> Więcej informacji: <https://manpages.debian.org/latest/apt/apt.8.html>.
+
+- Wyświetl krowę z jajem wielkanocnym:
```

Comment:
I don't think there is an "easter egg" equivalent in Polish. I looked at a couple of Polish websites and all of them just use the English version. [example](https://pl.wikipedia.org/wiki/Easter_egg)
```suggestion
> Easter egg `APT`.
> Więcej informacji: <https://manpages.debian.org/latest/apt/apt.8.html>.

- Wyświetl easter egga z krową:
```